src/main/java/com/epam/reportportal/jobs/clean/CleanAttachmentJob.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.reportportal.jobs.clean;
+
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+import org.apache.commons.lang3.math.NumberUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.Map;
+
+import static java.time.Duration.ofSeconds;
+
+/**
+ * @author <a href="mailto:pavel_bortnik@epam.com">Pavel Bortnik</a>
+ */
+@Service
+public class CleanAttachmentJob extends BaseCleanJob {
+
+	private final Logger LOGGER = LoggerFactory.getLogger(this.getClass());
+
+	private static final String MOVING_QUERY = "WITH moved_rows AS (DELETE FROM attachment WHERE project_id = ? AND creation_date <= ?::TIMESTAMP RETURNING *) INSERT INTO attachment_deletion (id, file_id, thumbnail_id, creation_attachment_date, deletion_date)  SELECT id, file_id, thumbnail_id, creation_date, now() FROM moved_rows;";
+
+	public CleanAttachmentJob(JdbcTemplate jdbcTemplate) {
+		super(jdbcTemplate);
+	}
+
+	@Scheduled(cron = "${rp.environment.variable.clean.attachment.cron}")
+	@SchedulerLock(name = "cleanAttachment", lockAtMostFor = "24h")
+	public void execute() {
+		moveAttachments();
+	}
+
+	private void moveAttachments() {
+		logStart();
```

Comment:
logStart() and logFinish() can be moved higher to execute() method

Reply:
Done